src/Core/Models/Business/OrganizationLicense.cs

```diff
@@ -145,7 +145,7 @@ public OrganizationLicense()
     /// </summary>
     /// <remarks>Intentionally set one version behind to allow self hosted users some time to update before
     /// getting out of date license errors</remarks>
-    private const int CURRENT_LICENSE_FILE_VERSION = 12;
+    public const int CURRENT_LICENSE_FILE_VERSION = 12;
```

Comment:
⛏️ This should follow coding standards if it's now used elsewhere so I'd assume it'll rename to `CurrentLicenseFileVersion`.